azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -533,29 +608,28 @@ def __init__(
         population: Optional[int] = None,
         sweeps: Optional[int] = None,
         beta: Optional[RangeSchedule] = None,
-        culling_fraction: Optional[float] = None,
     ):
-        """The constructor of Population Annealing Search solver.
+        """Constructor of the Population Annealing solver.
 
         Population Annealing Search solver for binary optimization problems
         with k-local interactions on an all-to-all graph topology with double
         precision support for the coupler weights.
 
-        This solver is CPU only, and not support parameter free now.
+        This solver is CPU only.
+        It currently does not support parameter-free invocation.
 
-        :param alpha:
-            ratio to trigger a restart, must be larger than 1
         :param seed:
-            specifies a random seed value.
-        :population:
-            size of target population, must be positive
+            Specifies the random number generator seed value.
         :param sweeps:
-            Number of monte carlo sweeps
+            Number of Monte Carlo sweeps. Must be positive.
+        :population:
+            Size of the population. Must be positive.
+        :param alpha:
```

Comment:
It looks like the order of the arguments in the function signature does not match the order of the arguments in the docstring. For consistency and ease of maintenance I'd recommend to match the order here.

Reply:
personally, I find this an unintuitive order for the arguments (and I would prefer there to be more correspondence when it comes to shared arguments with other solvers). I did not change the order however, since users might have code calling these constructors with positional arguments (i.e., would be a client-facing change).

Reply:
Agreed, though aside from it being a breaking change it still might be worth refactoring this in a separate PR; if you think it would be valuable to do so feel free to file an issue with your preferred argument ordering and we can discuss/look into it.